StellarApp/Protocol/Planet.swift

```diff
@@ -0,0 +1,220 @@
+                orbitNode.addChildNode(planetNode)
+                planetNodes.append(orbitNode)
+                
+            case .saturn:
+                let orbitNode = SCNNode.getRing(radius: 9.42, vector: (x: 0, y: 0, z: 0), color: .yellow)
+                let planetNode = SCNNode.getNode(name: "saturn", image: UIImage(named: "art.scnassets/saturn.jpg")!, radius: 0.86, vector: (x: 9.42, y: 0, z: 0), moveSpeed: 0.7, rotation: 0.2)
+                let saturnOrbitNode = SCNNode.getSaturnRing(radius: 0.9, vector: (x: 0, y: 0, z: 0), texture: UIImage(named: "art.scnassets/saturnRing.png")!)
+                orbitNode.addChildNode(planetNode)
+                planetNode.addChildNode(saturnOrbitNode)
+                planetNodes.append(orbitNode)
+                
+            case .uranus:
+                let orbitNode = SCNNode.getRing(radius: 12.58, vector: (x: 0, y: 0, z: 0), color: .yellow)
+                let planetNode = SCNNode.getNode(name: "uranus", image: UIImage(named: "art.scnassets/uranus.jpg")!, radius: 0.65, vector: (x: 12.58, y: 0, z: 0), moveSpeed: 0.5, rotation: 0.4)
+                orbitNode.addChildNode(planetNode)
+                planetNodes.append(orbitNode)
+                
+            case .neptune:
+                let orbitNode = SCNNode.getRing(radius: 15.98, vector: (x: 0, y: 0, z: 0), color: .yellow)
+                let planetNode = SCNNode.getNode(name: "neptune", image: UIImage(named: "art.scnassets/neptune.jpg")!, radius: 0.5, vector: (x: 15.98, y: 0, z: 0), moveSpeed: 0.4, rotation: 0.8)
+                orbitNode.addChildNode(planetNode)
+                planetNodes.append(orbitNode)
+                
+            case .pluto:
+                let orbitNode = SCNNode.getRing(radius: 18.65, vector: (x: 0, y: 0, z: 0), color: .yellow)
+                let planetNode = SCNNode.getNode(name: "pluto", image: UIImage(named: "art.scnassets/pluto.jpg")!, radius: 0.19, vector: (x: 18.65, y: 0, z: 0), moveSpeed: 0.4, rotation: 0.8)
+                orbitNode.addChildNode(planetNode)     
+                planetNodes.append(orbitNode)
+
+            }
+        }
+        return planetNodes
+    }
+}
+
+extension SCNNode {
+    static func deepScaleNode(node: SCNNode, scaleAction: SCNAction) {
+        node.runAction(scaleAction)
+        for child in node.childNodes {
+            deepScaleNode(node: child, scaleAction: scaleAction)
+        }
+    }
+    
+    static func deepScaleNode(node: SCNNode, scale: CGFloat) {
+        let scaleX = scale * CGFloat(node.scale.x)
+        let scaleY = scale * CGFloat(node.scale.y)
+        let scaleZ = scale * CGFloat(node.scale.z)
+        
+        let newScale = SCNVector3Make(Float(scaleX), Float(scaleY), Float(scaleZ))
+        node.scale = newScale
+        for child in node.childNodes {
+            deepScaleNode(node: child, scale: scale)
+        }
+    }
+    
+    
+    
+    static func getNode(name: String, image: UIImage,
```

Comment:
to make this closer to what the others indicate `getRing` etc. Maybe call this `getSphere`